diff --git a/node_modules/@cardinal/token-manager/dist/cjs/programs/tokenManager/constants.js b/node_modules/@cardinal/token-manager/dist/cjs/programs/tokenManager/constants.js
index 8ffb486..d23bf57 100644
--- a/node_modules/@cardinal/token-manager/dist/cjs/programs/tokenManager/constants.js
+++ b/node_modules/@cardinal/token-manager/dist/cjs/programs/tokenManager/constants.js
@@ -4,7 +4,7 @@ exports.CRANK_KEY = exports.PAYMENT_MANAGER_KEY = exports.TokenManagerState = ex
 const tslib_1 = require("tslib");
 const web3_js_1 = require("@solana/web3.js");
 const TOKEN_MANAGER_TYPES = (0, tslib_1.__importStar)(require("../../idl/cardinal_token_manager"));
-exports.TOKEN_MANAGER_ADDRESS = new web3_js_1.PublicKey("mgr99QFMYByTqGPWmNqunV7vBLmWWXdSrHUfV8Jf3JM");
+exports.TOKEN_MANAGER_ADDRESS = new web3_js_1.PublicKey(process.env.TOKEN_MANAGER_ADDRESS || "mgr99QFMYByTqGPWmNqunV7vBLmWWXdSrHUfV8Jf3JM");
 exports.MINT_COUNTER_SEED = "mint-counter";
 exports.MINT_MANAGER_SEED = "mint-manager";
 exports.TRANSFER_RECEIPT_SEED = "transfer-receipt";
diff --git a/node_modules/@cardinal/token-manager/dist/esm/programs/tokenManager/constants.js b/node_modules/@cardinal/token-manager/dist/esm/programs/tokenManager/constants.js
index ce35699..c0c625b 100644
--- a/node_modules/@cardinal/token-manager/dist/esm/programs/tokenManager/constants.js
+++ b/node_modules/@cardinal/token-manager/dist/esm/programs/tokenManager/constants.js
@@ -1,6 +1,6 @@
 import { PublicKey } from "@solana/web3.js";
 import * as TOKEN_MANAGER_TYPES from "../../idl/cardinal_token_manager";
-export const TOKEN_MANAGER_ADDRESS = new PublicKey("mgr99QFMYByTqGPWmNqunV7vBLmWWXdSrHUfV8Jf3JM");
+export const TOKEN_MANAGER_ADDRESS = new PublicKey(process.env.TOKEN_MANAGER_ADDRESS || "mgr99QFMYByTqGPWmNqunV7vBLmWWXdSrHUfV8Jf3JM");
 export const MINT_COUNTER_SEED = "mint-counter";
 export const MINT_MANAGER_SEED = "mint-manager";
 export const TRANSFER_RECEIPT_SEED = "transfer-receipt";
